src/pages/workspace/distanceRates/PolicyDistanceRateTaxReclaimableOnEditPage.tsx

```diff
@@ -0,0 +1,100 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React, {useCallback} from 'react';
+import {Keyboard} from 'react-native';
+import AmountForm from '@components/AmountForm';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapperWithRef from '@components/Form/InputWrapper';
+import type {FormOnyxValues} from '@components/Form/types';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useAutoFocusInput from '@hooks/useAutoFocusInput';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import {validateTaxClaimableValue} from '@libs/PolicyDistanceRatesUtils';
+import type {SettingsNavigatorParamList} from '@navigation/types';
+import AccessOrNotFoundWrapper from '@pages/workspace/AccessOrNotFoundWrapper';
+import type {WithPolicyOnyxProps} from '@pages/workspace/withPolicy';
+import withPolicy from '@pages/workspace/withPolicy';
+import * as Policy from '@userActions/Policy/Policy';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type SCREENS from '@src/SCREENS';
+import INPUT_IDS from '@src/types/form/PolicyDistanceRateTaxReclaimableOnEditForm';
+
+type PolicyDistanceRateTaxReclaimableOnEditPageProps = WithPolicyOnyxProps & StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.DISTANCE_RATE_TAX_RECLAIMABLE_ON_EDIT>;
+
+function PolicyDistanceRateTaxReclaimableOnEditPage({route, policy}: PolicyDistanceRateTaxReclaimableOnEditPageProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const {inputCallbackRef} = useAutoFocusInput();
+
+    const policyID = route.params.policyID;
+    const rateID = route.params.rateID;
+    const customUnits = policy?.customUnits ?? {};
+    const customUnit = customUnits[Object.keys(customUnits)[0]];
+    const rate = customUnit.rates[rateID];
+    const currency = rate.currency ?? CONST.CURRENCY.USD;
+    const currentTaxReclaimableOnValue = rate.attributes?.taxClaimablePercentage && rate.rate ? (rate.attributes.taxClaimablePercentage * rate.rate) / 100 : '';
+    const submitTaxReclaimableOn = (values: FormOnyxValues<typeof ONYXKEYS.FORMS.POLICY_DISTANCE_RATE_TAX_RECLAIMABLE_ON_EDIT_FORM>) => {
+        Policy.updateDistanceTaxClaimableValue(policyID, customUnit, [
+            {
+                ...rate,
+                attributes: {
+                    ...rate.attributes,
+                    taxClaimablePercentage: rate.rate ? (Number(values.taxClaimableValue) * 100) / rate.rate : undefined,
+                },
+            },
+        ]);
+        Keyboard.dismiss();
```

Comment:
What is the purpose of `Keyboard.dismiss()` here? Shouldn't it be dismissed automatically when you navigate away from the page and the input is blurred?

Reply:
@eVoloshchak I just tested again and realized it wasn't necessary